### DISCUSSION TITLE
Set custom trigger keys

### DISCUSSION THREAD

**Discussion by DavidSSL** [ggandor/leap.nvim#69] [Q&A] [answered]
I have tried to understand how to set custom keys so that I can use `Ctrl+;` instead of `s`. I've read the documentation but I don't understand the `<Plug>` part. Could you provide some guidance on how you would do this trigger re-mapping using Vim-plug? I have read [this](https://github.com/ggandor/leap.nvim/discussions/47) but that's not really helping me out either.

**Comment by ggandor** [comment, marked as answer]
There is an example at the bottom. Use `<Plug>(leap-foo)` as the right hand side, simple as that.

https://github.com/ggandor/leap.nvim/blob/a9d3232b178fddabb6660dc19abf40e05d6841b0/doc/leap.txt#L307-L335

> how you would do this trigger re-mapping using Vim-plug

`<Plug>` keys have nothing to do with vim-plug, the package manager. (You're not the first who is confused about this.) It is a native Vim feature, for aliasing stuff to be used in mappings.

**Reply by DavidSSL** [reply]
Thanks. For those like me who are just beginning, you will have to add the following to your `init.lua' file:

```lua
vim.keymap.set('n', 'm', '<Plug>(leap-forward-to)')
```

What this is saying is that in `n`ormal mode, when you press `m`, this will activate `<Plug>(leap-forward-to)`. I had to use `m` because `Ctrl+;` is not [possible](https://stackoverflow.com/questions/3101253/vim-mapping-ctrl).
